content/blog/blog-2020-07-01-kudo-dependencies.md

```diff
@@ -0,0 +1,93 @@
+---
+date: 2020-07-01
+---
+
+# KUDO Dependencies
+
+One of the most requested features (internally and externally) has been operator dependencies. We've experienced this pain ourselves: our own [Kafka](https://github.com/kudobuilder/operators/tree/master/repository/kafka) operator requires the user to manually [install Zookeeper](https://github.com/kudobuilder/operators/blob/master/repository/kafka/docs/v1.3/install.md) as a prerequisite. This sounds simple enough but can get out of hand quickly as our [Flink demo](https://github.com/kudobuilder/operators/tree/master/repository/flink/docs/demo/financial-fraud) shows.
+
+We've been thinking about the best way of introducing dependencies to KUDO for a while now. A few months ago we created [KEP-29](https://github.com/kudobuilder/kudo/blob/main/keps/0029-operator-dependencies.md) that aimed at solving at least part of the problems. Dependencies can be a complex topic. KEP-29 is not trying to boil the dependency ocean but rather limits itself to installation dependencies only, i.e. your operator instance and all its dependencies (including transitive ones) will be installed and/or removed as one unit.
+
+<!-- more -->
+
+## KudoOperator Task
+
+KUDO operators already have a mechanism to deal with installation dependencies called [plans, phases, and steps](plans.md) with serial or parallel execution strategy. This mechanism is already powerful enough to express any dependency hierarchy including transitive dependencies. As of KUDO 0.15.x, KUDO supports a `KudoOperator` task which allows you to specify a dependency on other operators. At the very least it requires the name of the operator to be installed. You can also provide an operator version (defaults to the latest one).
```

Comment:
it looks like `[plans, phases, and steps](plans.md)` was intended to be a link... however the link "plans.md" doesn't go anywhere... did you intend to?

Reply:
Good catch 👍